docs/user-guides/Connect-a-Wallet.md

```diff
@@ -0,0 +1,30 @@
+# Connect a Wallet
+
+## What You'll Need
+
+A compatible CSPR wallet: either [CasperLabs's Signer](https://www.notion.so/Signer-Guide-eb8c504d067344109351bc0f21ac2412) or [Ledger](https://www.ledger.com/). More wallets are planned for the future.
+
+## Sign In
+
+You can access the wallet Sign In screen by clicking `Sign In` from the top navigation menu. Then choose your Sign In option, either Signer or Ledger:
```

Comment:
Revise this to not offer Ledger as an option. It's not available yet. We can build this back in when the Ledger is available.

Reply:
:+1: Done.